src/octoclass.cpp

```diff
@@ -1165,6 +1234,60 @@ void OctoClass::PrintQueryInfo(octomap::point3d query,
     }
 }
 
+void OctoClass::InitializeMapToPathPlanningConfig() {
+    // Reset map first
+    tree_.clear();
+    tree_inflated_.clear();
+
+    // Get min/max coordinates of the map based on the config
+    const double x_min = std::min(path_planning_config_.map_x_corner1, path_planning_config_.map_x_corner2) + 0.001;
+    const double x_max = std::max(path_planning_config_.map_x_corner1, path_planning_config_.map_x_corner2) + 0.001;
+    const double y_min = std::min(path_planning_config_.map_y_corner1, path_planning_config_.map_y_corner2) + 0.001;
+    const double y_max = std::max(path_planning_config_.map_y_corner1, path_planning_config_.map_y_corner2) + 0.001;
```

Comment:
Why add the 0.001 offset?

Reply:
this is a good catch that I forgot to document! I will add a comment about this
The issue is that the octomap has the origin at `[0.0, 0.0, 0.0]`, and the nodes surrounding the origin (assuming a resolution of 0.1m) have positions at `[0.5, 0.5, 0.0]`, `[0.5, -0.5, 0.0]` , `[-0.5, 0.5, 0.0]`, and `[-0.5, -0.5, 0.0]`. So if we try to set a node that lives right at the margin between those nodes, the results are not always straightforward. Adding the `0.001` offset disambiguates this.
As an illustration, here is the map if I remove the `0.001` offset:
![image](https://user-images.githubusercontent.com/51488981/96923524-06a3e480-1477-11eb-9c18-416e519057f8.png)

I will add comments in code to clarify